app/Http/Controllers/SocialController.php

```diff
@@ -117,13 +126,16 @@ public function callback($provider)
      * @param $getInfo (response of Socialite->user())
      * @param $provider (String of login-provider)
      *
-     * @return user model
+     * @return RedirectResponse|Authenticatable model
```

Comment:
``RedirectResponse`` or ``Authenticatable``? What?

Yes, I see it in line 132 and 183, but thats weird... We should either return and Reponse or a Model.